src/GPIOInterface.php

```diff
@@ -19,12 +19,15 @@ public function getInputPin($number);
 
     /**
      * Get an output pin.
-     * 
-     * @param int $number The pin number
-     * 
-     * @return OutputPinInterface
+     *
+     * @param int  $number The pin number
+     *
+     * @param bool $activeLow
+     * @param null $defaultHigh
```

Comment:
`null` isn't really a data type - is this a boolean?

If so, should the default value be false?